lib/resources/aws/aws_flow_log.rb

```diff
@@ -0,0 +1,95 @@
+class AwsFlowLog < Inspec.resource(1)
+  name 'aws_flow_log'
+  supports platform: 'aws'
+  desc 'This resource is used to test the attributes of a Flow Log.'
+  example "
+    describe aws_flow_log('fl-9c718cf5') do
+      it { should exist }
+    end"
+
+  include AwsSingularResourceMixin
+
+  def to_s
+    "AWS Flow Log #{@flow_log_id}"
```

Comment:
If someone searches for a flow log by vpc_id or subnet_id, and it does not exist, @flow_log_id will be nil.  Could we add some logic to improve to_s in that case?